set.mm

```diff
@@ -641865,10 +641865,29 @@ D Fn ( I ... ( M - 1 ) ) /\
     ifexd.1 $e |- ( ph -> A e. V ) $.
     ifexd.2 $e |- ( ph -> B e. W ) $.
     $( Conditional operator existence.  (Contributed by SN, 26-Jul-2024.) $)
```

Comment:
I'd write
> Existence of the conditional operator (deduction form).

because it's easier to parse (with the current proposal, it could be read as "(conditional (operator existence))", and because it looks like it is more common (see `!SEARCH *ex exist / COMMENTS`)